ci/terraform/utils/bulk_test_user_delete_dynamo_access.tf

```diff
@@ -0,0 +1,57 @@
+data "aws_iam_policy_document" "user_profile_dynamo_delete_access" {
+  statement {
+    sid    = "AllowAccessToDynamoTables"
+    effect = "Allow"
+
+    actions = [
+      "dynamodb:BatchGet*",
+      "dynamodb:DescribeTable",
+      "dynamodb:Get*",
+      "dynamodb:Query",
+      "dynamodb:Scan",
```

Comment:
I'm not convinced we need `Scan` as we we'll `Query`ing on the index.

Reply:
Turns out we did, per your parallel comment on Slack, so keeping in (will resolve the other comment too which is the same lambda)